TestSolutionSpecflow/StepDefinitions/3. Items/ItemsStepDefinitions.cs

```diff
@@ -52,6 +53,13 @@ public void WhenTheUserSelectTheOptionInTheOptionsMenu(string selectedOption)
             _mainPage.ClickButton("ItemOptions", "Main Page", selectedOption);
         }
 
+        [When(@"The user set the Priority of the item to ""([^""]*)"" in the options menu")]
+        public void WhenTheUserSetThePriorityOfTheItemToInTheOptionsMenu(string priority)
+        {
+            _scenarioContext["Current Priority"] = priority;
+            _mainPage.ClickButton("PriorityOption", "Main Page", priority);
+        }
```

Comment:
Add a `Thread.Sleep(1000);` to wait for 1 second after clicking the priority button.
The page is not loading as fast as the test is going, that is why it is failing. It verifies the color before the page succesfully change it.